goat

```diff
@@ -151,9 +160,27 @@ link()
 		rm -- "$GOAT_PATH/$1"
 	fi
 
-	ln -fs -- "$targetpath" "$GOAT_PATH/$1"
+	$LN "$targetpath" "$GOAT_PATH/$1"
 }
 
+unix_ln()
+{
+	targetpath="$1"
+	destination="$2"
+	ln -fs -- "$targetpath" "$destination"
+}
+
+mingw_ln()
+{
+	targetpath="$1"
+	destination="$2"
+	#cmd "$destination" "$targetpath"
```

Comment:
Is this comment necessary? I feel like it doesn't tell the reader anything more than the next line with the actual command.